datafusion-cli/src/helper.rs

```diff
@@ -197,6 +204,19 @@ pub fn unescape_input(input: &str) -> datafusion::error::Result<String> {
     Ok(result)
 }
 
+/// Splits a string which consists of multiple queries.
+pub(crate) fn split_from_semicolon(sql: String) -> Vec<String> {
```

Comment:
You can probably avoid a copy here if you passed in a reference

```suggestion
pub(crate) fn split_from_semicolon(sql: &str) -> Vec<&str> {
```

Reply:
I cannot find a copy here. If pass the reference, I also need to copy it for `exec_and_print()`.